spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users/:id' do
+    let(:user) { User.create(name: 'John Doe') }
+
+    it 'renders the show template' do
+      get "/users/#{user.id}" # Directly specify the path
+      expect(response).to render_template(:show)
+    end
+
+    it 'has a 200 status code' do
+      get "/users/#{user.id}" # Directly specify the path
+      expect(response).to have_http_status(200)
+    end
+
+    it 'includes correct placeholder text' do
+      get "/users/#{user.id}" # Directly specify the path
+      expect(response.body).to include(user.name)
+    end
```

Comment:
- [ ] Great work writing the spec tests for the `show` action of the `Users` controller. Remember that one of the requirements for this project is to write request tests for all the actions in each of your controllers. In light of this, kindly write request spec tests for `index` action of the `Users` controller. This will guarantee that all requests for that action work as intended. 👍